src/app/server/FailSafeContext.cpp

```diff
@@ -89,7 +84,7 @@ void FailSafeContext::ScheduleFailSafeCleanup(FabricIndex fabricIndex, bool addN
     PlatformMgr().ScheduleWork(HandleDisarmFailSafe, reinterpret_cast<intptr_t>(this));
 }
 
-CHIP_ERROR FailSafeContext::ArmFailSafe(FabricIndex accessingFabricIndex, System::Clock::Timeout expiryLength)
+CHIP_ERROR FailSafeContext::ArmFailSafe(FabricIndex accessingFabricIndex, uint16_t expiryLengthSeconds)
```

Comment:
Why the signature change to use naming instead of a type to indicate the expiration length?

Reply:
To remove the direct dependency on SystemLayer which is the only place that has those types. If they were common types it would have been better

Reply:
Ugh, ok.  Makes sense, I guess.....